test/integration/functional_test.go

```diff
@@ -335,8 +335,9 @@ func validateDashboardCmd(ctx context.Context, t *testing.T, profile string) {
 
 	resp, err := retryablehttp.Get(u.String())
 	if err != nil {
-		t.Fatalf("failed to http get %q : %v", u.String(), err)
```

Comment:
can we have the URL back in to the 
failed to http get ... ?

Reply:
It's still there.